src/openrct2/ride/coaster/WoodenRollerCoaster.cpp

```diff
@@ -428,8 +428,9 @@ static void wooden_rc_track_paint_bb(paint_session* session, const sprite_bb_2*
     uint32_t railsImageId = bb->sprite_id_b | wooden_rc_get_rails_colour(session);
 
     PaintAddImageAsParent(
-        session, imageId, static_cast<int8_t>(bb->offset.x), static_cast<int8_t>(bb->offset.y), bb->bb_size.x, bb->bb_size.y,
-        static_cast<int8_t>(bb->bb_size.z), height + bb->offset.z, bb->bb_offset.x, bb->bb_offset.y, height + bb->bb_offset.z);
+        session, imageId, { static_cast<int8_t>(bb->offset.x), static_cast<int8_t>(bb->offset.y), height + bb->offset.z },
```

Comment:
Do we still need these static casts? I bet they were once needed to handle signed/unsignedness, but have been obsolete for a long time.

Reply:
We don't I missed them